classes/class-loader.php

```diff
@@ -342,6 +342,7 @@ public function enqueue_editor_scripts( $hook ) {
 			'features'        => [
 				'feature_engaging_networks' => $en_active,
 			],
+			'uses_cdn'        => get_option( 'sm_mode' ),
```

Comment:
It can be absent but also have different values like `cdn` `backup` or `disabled`, I guess we only want to aim at the `cdn` one, so `'cdn' === get_option( 'sm_mode' )` might be better ?

Reply:
I added the check for both `cdn` and `ephemeral`. I also added `stateless`, which seems to be the old name for `ephemeral`, in case a site would still need that.

Reply:
See https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/733#discussion_r766006227